client/settings/payment-methods-list/index.js

```diff
@@ -0,0 +1,205 @@
+/** @format */
```

Comment:
This is basically the old `client/payment-methods/index.js` component, but modified to accept a `methodIds` props - an array that represents the payment methods to render.
They could be BNPL methods on the good ol' UPE methods.